sdcflows/interfaces/bspline.py

```diff
@@ -288,116 +292,137 @@ def _run_interface(self, runtime):
 
 
 class _ApplyCoeffsFieldInputSpec(BaseInterfaceInputSpec):
-    in_data = InputMultiObject(
-        File(exist=True, mandatory=True, desc="input EPI data to be corrected")
-    )
+    in_data = File(exist=True, mandatory=True, desc="input EPI data to be corrected")
     in_coeff = InputMultiObject(
         File(exists=True),
         mandatory=True,
-        desc="input coefficients, after alignment to the EPI data",
+        desc="input coefficients as calculated in the estimation stage",
     )
-    in_xfms = InputMultiObject(
-        File(exists=True), desc="list of head-motion correction matrices"
+    fmap2data_xfm = InputMultiObject(
+        File(exists=True),
+        desc="the transform by which the target EPI can be resampled on the fieldmap's grid.",
+        xor="data2fmap_xfm",
+    )
+    data2fmap_xfm = InputMultiObject(
+        File(exists=True),
+        desc="the transform by which the fieldmap can be resampled on the target EPI's grid.",
+        xor="fmap2data_xfm",
+    )
```

Comment:
Seems confusing making these MultiObjects, since the first is always being used and the rest discarded - and we should specify that these need to be in ITK format.

Reply:
This is for compatiblity with the nipype interface. We probably want not to drop linear transforms after the first, although it feels like a rabbit hole.

So, rather than operating here, I think we want to open an issue to think about how a list here should be managed later.